guides/cookbook.md

```diff
@@ -1297,6 +1667,42 @@ writer.close();
 ```
 </div>
 
+<div class="tabpane Python" markdown="1">
+Regardless of the writer API used, the same CSV conversion method can be used.
+
+```python
+data = ... # An object suporting `readlines` over the CSV data.
```

Comment:
let's pick one and show it so folks have an example that works

Reply:
Done.